Source/JavaScriptCore/jit/ExecutableMemoryHandle.h

```diff
@@ -32,6 +32,13 @@
 #include <wtf/MetaAllocatorHandle.h>
 #endif
 
+#if !USE(SYSTEM_MALLOC)
+#include <bmalloc/BPlatform.h>
+#if BENABLE(LIBPAS) && (OS(DARWIN) || OS(LINUX))
+#define USE_LIBPAS_JIT_HEAP 1
```

Comment:
I think that USE_LIBPAS_JIT_HEAP may be a misnomer, shouldn't it be ENABLE? 

```
/* USE() - use a particular third-party library or optional OS service */
```

Reply:
Seems reasonable.